test/model/updateMany.test.ts

```diff
@@ -177,6 +177,58 @@ test('should update many entities with primitive values', () => {
   updateMultiUsers.forEach((user) => expect(user.role).toEqual('Admin'))
 })
 
+test('should update many entities to and from null values', () => {
+  const db = factory({
+    user: {
+      id: primaryKey(datatype.uuid),
+      firstName: name.findName,
+      role: nullable<string>(() => null),
+    },
+  })
+  db.user.create({
+    firstName: 'Joseph',
+    role: 'Auditor',
+  })
+
+  db.user.create({
+    firstName: 'John',
+    role: 'Auditor',
+  })
+
+  db.user.create({
+    firstName: 'Jack',
+    role: 'Writer',
+  })
+
+  const oldAuditors = db.user.updateMany({
+    where: {
+      role: {
+        equals: 'Auditor',
+      },
+    },
+    data: {
+      role: null,
+    },
+  })!
+
+  expect(oldAuditors).toHaveLength(2)
```

Comment:
Using a `prev` prefix is a nice way to denote a previous state of a value. [See the naming cheatsheet](https://github.com/kettanaito/naming-cheatsheet#prevnext). 


```diff
-const oldAuditors = value
+const prevAuditors = value
```

Unless you mean "old of age" when saying "old" of course :D

Reply:
Oh I hadn't seen that naming cheatsheet, I'll have a read of it now 👍